Node/src/TeamsMessage.ts

```diff
@@ -41,19 +41,54 @@ export enum MentionTextLocation {
   AppendText
 }
 
+export class MentionEntity {
+  type: string;
+  mentioned: any;
+  text: string;
+}
+
+export class UserMention extends MentionEntity
+{
+    constructor(userId: string, name: string) {
+      super();
+      this.type = 'mention';
+      this.text = '<at>'+name+'</at>';
+      this.mentioned = {
+        'id' : userId,
+        'name' : name,
+        'type': 'user'
+      };
+    }
+}
+
+export class ChannelMention extends MentionEntity
+{
+    constructor(channelId: string, name: string) {
```

Comment:
Shouldn't this just be (channel: ChannelInfo)
(Is there a use case for overriding the channel name?)